contents/docs/tutorials/cohorts.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Analyzing User Behavior with Cohorts
+sidebar: Docs
+showTitle: true
+---
+
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 7 minutes ☕☕_</small>
+
+<br />
+
+<span class="larger-image">
+
+![Cohorts banner image](../../images/tutorials/banners/cohorts.png)
+
+</span>
+
+<br />
+
+Each user uses your product slightly differently from the next, and some user groups might use your app in a completely different way from others.
+
+This happens for a variety of reasons, including software, user profiles, accesibility needs, culture, understanding of the product, individual goals, among many other variables.
+
+As such, an established technique for performing analytics is cohort analysis. When doing cohort analysis, users are split into different groups according to certain characteristics, and you can then derive metrics for each specific group (cohort), rather than only performing your analytics on the entire collection of users.
+
+This is extremely useful for comparing metrics across cohorts in order to determine areas for improvement, and maybe even product-market fit.
+
+Here are some examples:
+
+**Segmenting by browser**
+
+Allows you to determine if your UX is particularly poor on certain browsers. This can be indicated by a lower retention, bad conversion on your funnels, or a lack of certain events being captured. 
+
+**Segmenting by audience type**
+
+When you sign up, websites like GitHub will ask you about your goals with the platform, as well as your level of experience in a certain field. This information can be used to group users, and you can then see which users are best suited for your platform. For instance, in GitHub's case, you'd expect retention to be higher amongst people who use GitHub for their jobs instead of hobbyists.
+
+**Segmenting by engagement level**
+
+Another useful way of grouping users is by their engagement with your product. How do power users use your product when compared to free trial users? 
+
+### Cohorts in PostHog
+
+User group segmentation for cohort analysis is built into PostHog via our cohorts feature.
+
+Cohorts let you group users by properties in common, as well as by shared actions over a time period.
```

Comment:
we should mention in the actions tutorial #540 that these are *necessary* (since you can't just use events) to create cohorts of users based on previous behavior (and this should also go in the main actions page of the docs if it's not there), as this is an important point.